src/components/basic_commands.py

```diff
@@ -1904,13 +1908,17 @@ def le_read_local_resolvable_address(transport, idx, PeerIdAddrType, AVal, to):
     cmd = struct.pack('<HHHB6B', Commands.CMD_LE_READ_LOCAL_RESOLVABLE_ADDRESS_REQ, 9, HCICommands.BT_HCI_OP_LE_READ_LOCAL_RPA, PeerIdAddrType, *AVal);
     transport.send(idx, cmd);
     
-    packet = transport.recv(idx, 11, to);
-    
-    if ( 11 != len(packet) ):
-        raise Exception("LE Read Local Resolvable Address command failed: Response too short (Expected %i bytes got %i bytes)" % (11, len(packet)));
+    packet = transport.recv(idx, 5, to);
+    RespCmd, RespLen, status = struct.unpack('<HHB', packet);
+
+    if ( status == 0 ):
```

Comment:
This causes amongst others BV-14-C to fail when running against the Zephyr LL with status equals 2 (unknown connection identifier IIRC).
Further investigation is required on why.

Reply:
@kruithofa Are you investigating this?

Reply:
My bad, the full name is LL/DDI/SCN/BV-14-C.

Yes, I am investigating this. I had a look at it, and need to discuss further with Aurora

Reply:
The offending commit has been taken out from this PR, and issue #21 was created